### DISCUSSION TITLE
5 Node cluster Question where RMQ node keep ask to stop

### DISCUSSION THREAD

**Discussion by Kprash10** [rabbitmq/rabbitmq-server#10117] [Other]
I am using 5 node cluster where i make sure cookies and cert must followed same as node one Still if i use this with my application Queue is not looks mirror not sure that cause an issue but after i build this cluster start seeing issue more often about node 
![image](https://github.com/rabbitmq/rabbitmq-server/assets/153684155/d423d4d1-1d45-4ec7-a226-38308de8c677)

  * We saw DOWN from rabbit@<ServerA>
2023-12-12 12:56:01.768000-07:00 [error] <0.27066.58>  * We can still see rabbit@SErverb1 which can see rabbit@ServerA
2023-12-12 12:56:01.768000-07:00 [error] <0.27066.58>  * pause_minority mode enabled
2023-12-12 12:56:01.768000-07:00 [error] <0.27066.58> We will therefore pause until the *entire* cluster recovers
2023-12-12 12:56:01.768000-07:00 [warning] <0.27066.58> Cluster minority/secondary status detected - awaiting recovery
2023-12-12 12:56:01.768000-07:00 [info] <0.32230.68> RabbitMQ is asked to stop...

 Mnesia('rabbit@SErverA'): ** ERROR ** mnesia_event got {inconsistent_database, running_partitioned_network, 'rabbit@SErverA'

Not sure if this related to network because i never saw any network drop 

Any guidance to troubleshoot this issue 

RMQ version : 3.11.5.2

**Comment by michaelklishin** [comment]
```
We can still see rabbit@SErverb1 which can see rabbit@ServerA
```

is a partial partition. C cannot communicate with A but can with B, and B can communicate with A.

During events when queues sync from replicas (a newly added node, a node restart, a change in replication factor) this may suggest that this specific pair of nodes overloads its network link.

FYI, classic mirrored queues have [long been deprecated](https://rabbitmq.com/ha.html). [Use quorum queues](https://rabbitmq.com/quorum-queues.html). They are much more efficient in terms of data transfers (where possible, obviously a freshly added replica would still sync everything) and are much safer.

**Reply by Kprash10** [reply]
Thank you for your brief detail about issue also respect your time to response. 
Here what my Queue looks like :
![image](https://github.com/rabbitmq/rabbitmq-server/assets/153684155/ecce4cba-8ce5-4b3f-b57f-77af5ff3f88d)
You can see we are using both Quorum and classic queue both . 
![image](https://github.com/rabbitmq/rabbitmq-server/assets/153684155/f6eea92a-5251-4a68-9413-d5d710e8043b)
This is mainly discovered by 5 node cluster. 

Can i relate this issue with network issue or cluster issue?
